omgeo/services/__init__.py

```diff
@@ -658,9 +658,11 @@ def _geocode(self, pq):
                 query = dict(query, searchExtent=pq.viewbox.to_esri_wgs_json())
         else: # single-line
             method = 'find'
+            magic_key = pq.key if hasattr(pq, 'key') else ''
```

Comment:
Did we verify that passing an empty string results in the same behavior as omitting the query argument entirely?